app/(main)/league/[leagueId]/entry/[entryId]/week/WeekTeams.tsx

```diff
@@ -59,25 +59,34 @@ const WeekTeams = ({
   >
     {schedule.map((scheduledGame) => (
       <div
-        className="grid w-full grid-cols-2 gap-4 pb-8"
+        className="grid w-full grid-cols-[1fr_auto_1fr] gap-4 pb-8"
+        style={{ direction: 'rtl' }}
```

Comment:
Why is this hear? This is typically used to support languages that are written right to left, unlike english which is written left to right.

Reply:
it was shashi's suggestion. he wanted to flip everything in the grid layout so that the away teams were on the left and home teams were on right. because the data always rendered vise versa

Reply:
Okay that's what I thought it was doing. I wonder if CSS the `order` [property](https://developer.mozilla.org/en-US/docs/Web/CSS/order) would be more applicable here? @shashilo what are your thougths? I'm also worried about the a11y implication here.

Reply:
This is correct. The data receive from ESPN is reversed. We don't alter the source data, we have to work with it.